.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+**What this PR does / why we need it**:
```

Comment:
I definitely think something to refer to the documentation and testing changes specified in the initial Github issue (if a feature) to signify that they have been completed would be worthwhile.